### PR TITLE
admin snapshot rankings

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Update snapshotRanking component to render with data, plus some additional smaller UI/UX revisions (mostly handling `dropdownInput` interactions).

## WHY
This is the last significant piece of the frontend we need in place before this feature can ship.

## HOW
Just CSS and React updates.

### Screenshots
Screen Shot 2023-06-07 at 11 49 56 AM
Screen Shot 2023-06-07 at 11 49 45 AM
Screen Shot 2023-06-07 at 11 49 33 AM
Screen Shot 2023-06-07 at 11 48 33 AM

### Notion Card Links
https://www.notion.so/quill/Usage-snapshot-rankings-29c7075a09634df7931a44b7bab78c69?pvs=4

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/__tests__/snapshotRanking.test.tsx [hunk 1]
```diff
@@ -6,6 +6,20 @@ import { timeframes, grades, schools, } from './data'
 import { SMALL, NEGATIVE, POSITIVE, MEDIUM, } from '../shared'
 import SnapshotRanking from '../snapshotRanking';
 
+const data = [
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/__tests__/snapshotRanking.test.tsx:9]: Oh, I really like making a snapshot for stubbed data like this.

File: services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ interface DropdownInputState {
 
 const KEYDOWN = 'keydown'
 const MOUSEDOWN = 'mousedown'
+const TOUCHSTART = 'touchstart'
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx:50]: I assume that this is an accessibility enhancement?

**Review by anathomical — APPROVED** [review]
Looks good to me!

File: services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx (hunk 2 above)
Comment (emilia-friedberg) [reply services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx:50]: @anathomical responsive design! I swear it used to work, but for whatever reason the `dropdownInput` stopped responding to touchevents on mobile.

Comment (anathomical) [reply services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx:50]: Probably some update to the APIs that they forgot to tell everyone about before making.

**Review by eadams17 — APPROVED** [review]
Everything LGTM!

**Pull request merged**